docs/_static/auth/iam.yaml

```diff
@@ -562,6 +562,290 @@ paths:
+                  id_token_types_supported:
+                    type: array
+                    items:
+                      type: string
+                    description: | 
+                      A JSON array of strings containing the list of ID Token types supported by the OP, the default value is attester_signed_id_token. The ID Token types defined in this specification are:
+                      subject_signed_id_token: Self-Issued ID Token, i.e. the ID Token is signed with key material under the End-User's control.
+                      attester_signed_id_token: the ID Token is issued by the party operating the OP, i.e. this is the classical ID Token as defined in [OpenID.Core].
+        default:
+          $ref: '../common/error_response.yaml'
+  /iam/oid4vci/dcr/{sessionId}/authorize:
+    get:
+      operationId: dcrOid4vpAuthorize
+      summary: Authorize in the Dynamic Credential Request OID4VP flow.
+      tags:
+        - dcr
+        - oid4vp
+      parameters:
+        - name: sessionId
+          in: path
+          required: true
+          description: The sessionId of the callback request.
+          schema:
+            type: string
+            example: 55d7a35d-d7bf-436f-80f7-3fef4077f8a8
+        - name: request
+          in: query
+          required: true
+          description: | 
+            Should be a signed request object as JWT with the following required fields:
+              - response_type : vp_token
+              - client_id : did value
+              - client_id_scheme : "did"
+              - redirect_uri: the uri to redirect to
+              - state: state 
+              - nonce: unique 
+              - presentation_definition or presentation_definition_uri
+              - user_hint
+          schema:
+            type: string
+      responses:
+        '302':
+          description: |
+            The user-agent is redirected to the redirectURL submitted at the start-oid4vci-issuance request.
+          headers:
+            Location:
+              schema:
+                type: string
+                format: uri
+  /iam/oid4vci/{sessionId}/callback:
+    get:
+      operationId: callbackOid4vciIssuance
+      summary: Callback for the Oid4VCI flow.
+      description: |
+        Initiates an Oid4VCI flow to request an VC from a Credential Issuer.
+
+        error returns:
+        * 400 - one of the parameters has the wrong format or an OAuth error occurred
+        * 412 - the organization wallet does not contain the correct credentials
+        * 503 - the authorizer could not be reached or returned an error
+      tags:
+        - auth
+      parameters:
+        - name: sessionId
+          in: path
+          required: true
+          description: The sessionId of the callback request.
+          schema:
+            type: string
+            example: 55d7a35d-d7bf-436f-80f7-3fef4077f8a8
+        - name: code
```

Comment:
also add `state`, `error` and `error_description` params.

Reply:
Done

Reply:
Check, done. Also added error handling code.